comp120_1507866.tex

```diff
@@ -0,0 +1,54 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+
+% Please include a clear, concise, and descriptive title
+\title{Reflective Report}
+
+% Please do not change the subtitle
+\subtitle{COMP120 - Reflective Report}
+
+% Please put your student number in the author field
+\author{1507866}
+
+\begin{document}
+	
+\maketitle
+	
+\abstract{This report will look over the last semester and reflect on what I believe to have been my three main weaknesses}.
+	
+\section{Introduction}
+After reviewing my weekly reports for this semester I feel that my three main weaknesses are my time management, problems solving and critiquing skills.
+
+\section{Time Management}
+Time management is the skill that needs the most improvement. I have found when I have many assignments set it can be overwhelming. It would be better to plan and assign time to each one.
+
+Currently I assign time to an assignment and either not do enough work or focus too much on other assignments. This is a problem as it means I am not spending enough time on some assignments.
+
+Previously I have been using Trello boards to plan my Kivy app. In that particular assignment I found Trello helpful. I think a possible way to improve my time management would be to use Trello boards for more assignments and make them more detailed. The use of Trello boards also links to using sprint plans. For future assignments I am aiming to use more sprint plans. I can plan what needs to be done in an assignment and then use a sprint plan to stay on track. 
+
+
+\section{Problem Solving}
+Another weakness from the last semester is my problem solving abilities. This was initially a problem for the SpaceChem worksheet. On many of the levels I found that my solutions were poor in both the number of elapsed cycles and symbols used.
+
+Problems solving also effects my coding abilities. For both the university course and afterwards, improving on these skills should help me find better solutions when programming.
```

Comment:
*affects* my *programming* abilities. 

Be more specific about your future career goals. If it's to be a programmer, then state this clearly.